src/components/garagePromotionCard.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, MouseEvent } from "react"
```

Comment:
this comment is regarding the filename - do we imagine ever using this component for something else than for the garage promotion? then I'd rename it to something more generic already now

Reply:
Yes you're right. I will find more generic names for both  file and props.